java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -218,74 +224,56 @@ public static void assertPartialColumnsAreEqual(HostColumnVectorCore expected, l
                 "Column " + colName + " Row " + tableRow);
             break;
           case LIST:
-            assertListColumnsEquals(expected, cv, colName, enableNullCheck);
+            HostMemoryBuffer expectedOffsets = expected.getOffsets();
```

Comment:
> We should ideally check the length of the list instead of the exact offsets.

That is what I intended for this code to do.  IIUC LIST offsets are always row indices, not data indices, into the child column.  The code computes the number of rows in the child column for this row in the expected and actual parent list columns and compares the child row count (i.e.: the length of the list) between the two.

Reply:
Sorry you are right.  This is why I need to go over the code again.